src/scss/config/_color.scss

```diff
@@ -137,9 +137,6 @@ $_ccs: (
 ///   See map for valid tokens.
 /// @return {*} -
 ///   Value from `$ccs` map, based on given key
-/// @example
-///   @use '../config/color';
-///   $prime-hue: color.ccs('prime');
```

Comment:
@mirisuzanne This was breaking Sassdoc (likewise the other SCSS examples) -- I think because https://github.com/oddbird/sassdoc-theme-herman/issues/341